### ISSUE TITLE
i think it's a bug

### ISSUE THREAD

**Issue by AlexDAM** [toolbox4minecraft/amidst#301]
so i downloaded the latest version of amidst the 4.22 that suports minecraft 1.11 but when i open it it open a window and it says Please select a Minecraft Profile: Forge                     not found and forge          1.8.9 found should it show all the versions or at least the 1.11 version of minecraft i have deleted and download amidst 3 times but it's the same what should i do???

**Comment by stefandollase** [comment]
Please read [Using Amidst](https://github.com/toolbox4minecraft/amidst/wiki/Using-Amidst). If this does not help you, please describe your problem more clearly. Also, please use punctuation. Right now, it is really hard to follow your description.

**Comment by AlexDAM** [comment]
does amidst work with cracked clients i can't buy minecraft so i am using the TLauncher to play i have created a profile there but it dosn't show me the profile or any versions yet maybe amidst dosn't work with any other clinets???

**Comment by stefandollase** [comment]
Indeed, we only support the official Minecraft launcher.
